### ISSUE TITLE
Fusionpbx on FreeBSD git clone to pseudo-random location

### ISSUE THREAD

**Issue by tonyhain** [fusionpbx/fusionpbx#2534]
resources/switch/source-*.sh target git clone to /usr/src
as that script is buried, it is not obvious that it even happened, particularly if you aren't looking in /usr/src. It would have been better if the initial install.sh had found the path for fusionpbx src
ie:   pwd && cd ../.. && pwd  then parked any subsequent source in parallel with fusionpbx src so it would be seen.

**Comment by markjcrane** [comment]
This issue belongs here https://github.com/fusionpbx/fusionpbx-install.sh/issues.

**Comment by markjcrane** [comment]
The FusionPBX source is PHP, HTML, CSS, Javascript and LUA script and its put in the web directory at this point and therefore putting FreeSWITCH source together with it in the web directory is not a good plan. FusionPBX code currently doesn't need to be compiled so we don't pull the code to some other location at least currently. The conversation needs to be taken directly you can call FusionPBX support number at +1 208-906-8227 to reach me. Or you can continue this further at.
https://github.com/fusionpbx/fusionpbx-install.sh/issues
